src/util/StreamerMode.hpp

```diff
@@ -4,7 +4,11 @@
 
 namespace chatterino {
 
-enum StreamerModeSetting { Disabled = 0, Enabled = 1, DetectObs = 2 };
+enum StreamerModeSetting {
+    Disabled = 0,
+    Enabled = 1,
+    DetectStreamingSoftware = 2
```

Comment:
Using this enum literal will make git diffs of any potential future changes cleaner
```suggestion
    DetectStreamingSoftware = 2,
```